src/routes/project/TaskRow.svelte

```diff
@@ -45,7 +45,12 @@
 	}
 </script>
 
-<div class="task {taskStatusClasses[task.status]}" style="opacity: {task.isDormant ? 0.3 : 1};">
+<div
+	class="task {taskStatusClasses[task.status]}"
+	style="opacity: {task.isDormant ? 0.3 : 1};{store.mode === 'user' &&
+		task.isDormant &&
+		'display: none'}"
```

Comment:
Well, this is getting convoluted. Let's use the [class](https://learn.svelte.dev/tutorial/classes) directive.
```suggestion
	class:task--dormant={task.isDormant}
	class:task--dev-mode={$mode == 'developer'}
```